pkg/apis/infinispan/v1/infinispan_types.go

```diff
@@ -6,7 +6,9 @@ import (
 
 // InfinispanSpec defines the desired state of Infinispan
 type InfinispanSpec struct {
-	Size int32 `json:"size"`
+	Size        int32  `json:"size"`
+	ClusterName string `json:"clustername"`
```

Comment:
This shouldn't be here, it was removed by ba950d6860bae1bcaa795cfee407f6ff8175e874

Reply:
weird, fixed